pages/linux/aura.md

```diff
@@ -0,0 +1,36 @@
+# aura
+
+> The Aura Package Manager: A secure, multilingual package manager for Arch Linux and the AUR.
+> More information: <https://github.com/fosskers/aura>.
+
+- Search for packages from the repos and AUR:
```

Comment:
I can't find the `--both` flag on the GitHub page, but I assume it enables the official repos too, right?

Reply:
Yes all the repos in /etc/pacman.conf
And it's in the man page.

Reply:
```suggestion
- Search for packages from the official repositories and AUR:
```